backend/tessend/tessend.go

```diff
@@ -345,27 +391,83 @@ func (t *Tessend) collect(now int64) *Data {
 			Version: version.Semver(),
 			License: wrapper.Value.License,
 		},
-		Metrics: corev2.Metrics{
-			Points: []*corev2.MetricPoint{
-				&corev2.MetricPoint{
-					Name:      "entity_count",
-					Value:     entityCount,
-					Timestamp: now,
-				},
-				&corev2.MetricPoint{
-					Name:      "backend_count",
-					Value:     backendCount,
-					Timestamp: now,
-				},
+	}
+
+	return data
+}
+
+// getTessenMetrics populates the data payload with # backends/entities.
+func (t *Tessend) getTessenMetrics(now int64, data *Data) {
+	var entityCount, backendCount float64
+
+	// collect entity count
+	entities, err := t.store.GetEntities(t.ctx, &store.SelectionPredicate{})
```

Comment:
It would be best to avoid getting the entire list of entities here. Can you add an `EntityCount` method to the store using the `clientv3.WithCountOnly` option?

We'll need this for another component also, so we can get two birds stoned at once. :bird: :evergreen_tree:

Reply:
Yeah, I filed https://github.com/sensu/sensu-go/issues/2882 back when we discovered that improvement. I suppose I can add that to this PR.

Reply:
Alright, added a `Count()` function to the generic store.. utilizing it here.